src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/projectcreation/view/fragments/SelectBook.kt

```diff
@@ -1,27 +1,35 @@
 package org.wycliffeassociates.otter.jvm.app.ui.projectcreation.view.fragments
 
 import javafx.geometry.Pos
+import org.wycliffeassociates.otter.jvm.app.UIColorsObject.Colors
 import org.wycliffeassociates.otter.jvm.app.ui.projectcreation.viewmodel.ProjectCreationViewModel
+import org.wycliffeassociates.otter.jvm.app.ui.projecthome.ProjectHomeView
 import tornadofx.*
 
-class SelectBook() : View() {
+
+class SelectBook : View() {
```

Comment:
Seems like there are two types of select views: one for major sections like Bible/tW/tN and anthology, and the other for the specific project (which is a book in the case of the Bible). Is there anyway we could consolidate things into `SelectType` and `SelectProject` views? Just thinking out loud to see if that might work, but I don't know much about the Wizard component

Reply:
Do you mean like a generic Select<T> view?